app/views/applications/process/summary.html.slim

```diff
@@ -17,4 +17,4 @@ header
     .row.collapse.medium-10.large-8
       =render(partial: 'shared/remission_type', locals: { source: @result })
 
-=link_to 'Complete processing', application_confirmation_path(@overview.application.id), class: 'button primary'
+=link_to 'Complete processing', application_summary_save_path(@overview.application.id), class: 'button primary'
```

Comment:
Related to the comment in `routes.rb`, this should be a form, even though an "empty one". A simple 

```
= form_for @application, url: :summary_save_path, method: :post, html: { autocomplete: 'off' } do |f|
  = f.submit 'Complete processing', :class => 'button primary'
``` 

We don't need form object for it or anything.